### PR TITLE
Cockpit 281

### PR DESCRIPTION
**Pull request by garrett**
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-281.html

### CODE REVIEW

File: _posts/2022-12-01-cockpit-281.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Add style switcher
+
+Due to browsers not always picking up the correct system dark mode and users preferring to override the system dark mode Cockpit now has a style switcher in the session menu.
+
+![Screenshot from 2022-12-01 15-09-39]({{ site.baseurl }}/images/281-add-style-switcher.png)
+
+## Disallow root login by default
+On all operating systems, logging in as root with a username/password via SSH has been disallowed for a long time. Cockpit now also disallows it by default on new installations, but still allows it on upgrades for backwards compatibility. You can configure this in `/etc/cockpit/disallowed-users`.
+
+## Autodetect image type when adding a disk via a custom path
```
Comment (KKoukiou) [inline _posts/2022-12-01-cockpit-281.md:26]: Missing 'Machines' header prefix. We used to always add the prefix for all pages. Isn't this the case now?

Comment (garrett) [reply _posts/2022-12-01-cockpit-281.md]: It is. It's supposed to be autodetected and applied to the PR's title, but when someone sets a custom header, that's used instead (so it has to be manually added to manual headers).

Fixed in the PR; updated in the notes here.

**Comment by skobyda** [comment]
Updated bodhi links and summary, I think now it's ready to land

**Comment by garrett** [comment]
I've squashed and merged it. We could always add fix-ups afterward, but I also think it's ready to go.

**Pull request merged**